pwnlib/tubes/ssh.py

```diff
@@ -1846,8 +1846,8 @@ def _init_remote_platform_info(self):
 
         ::
 
-            {'distro': 'Ubuntu\n',
-             'distro_ver': '14.04\n',
+            {'distro': 'Ubuntu\\n',
+             'distro_ver': '14.04\\n',
```

Comment:
Please use `r'''` / `r"""` a few lines above instead of duplicating backslashes.

Reply:
Changed as requested.